utils/utils.py

```
@@ -198,13 +199,11 @@ async def run_orion(
         "es_benchmark_index": es_benchmark_index,
     }
 
-    # Log env
-    print(f"Env: version={version}, es_metadata_index={es_metadata_index}, es_benchmark_index={es_benchmark_index}")
+    logger.debug("Env: version=%s, es_metadata_index=%s, es_benchmark_index=%s", version, es_metadata_index, es_benchmark_index)
     result = await run_command_async(command, env=env, cwd="/tmp")
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)